pkg/seven/apisix/route.go

```diff
@@ -49,7 +49,7 @@ func FindCurrentRoute(route *v1.Route) (*v1.Route, error) {
 		}
 
 	}
-	return nil, fmt.Errorf("NOT FOUND")
+	return nil, utils.NotFound
```

Comment:
What about`ErrNotFound`? It a convention that all error object should be named with prefix `Err`.

Reply:
Done